templates/cloudflare-pages/package.json

```
@@ -15,15 +15,15 @@
     "@remix-run/cloudflare-pages": "*",
     "@remix-run/react": "*",
     "cross-env": "^7.0.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@cloudflare/workers-types": "^3.18.0",
     "@remix-run/dev": "*",
     "@remix-run/eslint-config": "*",
-    "@types/react": "^17.0.52",
-    "@types/react-dom": "^17.0.18",
+    "@types/react": "^18.0.26",
+    "@types/react-dom": "^18.0.9",
```

Let's keep these the same as the other templates

```suggestion
    "@types/react": "^18.0.25",
    "@types/react-dom": "^18.0.8",
```